### PR TITLE
Update flavor cache as encountered during 2023.10 invoicing

### PR DESCRIPTION
**Pull request by knikolla**
This PR updates the flavor cache with the flavors that were encountered during the October 2023 invoicing.

### CODE REVIEW

File: etc/flavors_cache.json [hunk 1]
```diff
@@ -341,5 +341,82 @@
...
+    {
+        "id": 332,
+        "name": "gpu-su-v100.1",
+        "vcpus": 48,
+        "memory": 196608,
+        "storage": 20
+    },
+    {
+        "id": 335,
+        "name": "gpu-su-v100.1m",
```
Comment (joachimweyl) [inline etc/flavors_cache.json:375]: @Milstein I thought we got rid of the v100.1m?

File: etc/flavors_cache.json [hunk 2]
```diff
@@ -341,5 +341,82 @@
...
+        "vcpus": 48,
+        "memory": 196608,
+        "storage": 20
+    },
+    {
+        "id": 368,
+        "name": "test-gpu-su-k80.4",
```
Comment (joachimweyl) [inline etc/flavors_cache.json:382]: @Milstein what is the test-gpu-su-k80.4?

File: etc/flavors_cache.json [hunk 3]
```diff
@@ -341,5 +341,82 @@
...
+    {
+        "id": 426,
+        "name": "test2-gpu-su-k80.4",
+        "vcpus": 24,
+        "memory": 118784,
+        "storage": 20
+    },
+    {
+        "id": 356,
+        "name": "Unknown test-gpu-su-k80.4",
```
Comment (joachimweyl) [inline etc/flavors_cache.json:403]: @Milstein what are Unknown flavors?

File: etc/flavors_cache.json (hunk 1 above)
Comment (knikolla) [reply etc/flavors_cache.json:375]: The flavors cache will show the list of deleted flavors as well, so a flavor being here doesn't mean it still exists. We need to keep track of deleted flavors.

**Review by joachimweyl — APPROVED** [review]

File: etc/flavors_cache.json (hunk 3 above)
Comment (knikolla) [reply etc/flavors_cache.json:403]: @joachimweyl *Unknown* flavors are flavors that were both created and deleted within the same billing cycle and therefore never appeared in a database dump to allow access to authoritative information. I had to piece together what they might be about from tangential information like what hypervisor the VM using that flavor was running on, what other flavors are running on that hypervisor, what is the name of the VM and of other VMs running on that hypervisor.

**Pull request merged**